### DISCUSSION TITLE
[Feature Request] Add Optional Edit Icon at the Top of Pages

### DISCUSSION THREAD

**Discussion by VaclavElias** [dotnet/docfx#9247] [General]
**Is your feature request related to a problem? Please describe.**
When the documentation is lengthy (e.g., [Create a Project](https://doc.stride3d.net/latest/en/manual/get-started/create-a-project.html)), a visitor might not realize that there is an "Edit" link at the bottom of the page. Although the link should also remain at the bottom for convenience, having an additional option at the top could be useful.

**Describe the solution you'd like**
I suggest adding an optional, opt-in "Edit" icon at the top of the page, perhaps next to the breadcrumb navigation, similar to what is seen in Microsoft Docs.

![image](https://github.com/dotnet/docfx/assets/4528464/89182697-c056-421b-b769-1f29564fd5a9)

Example of the icon location, I am not sure about the exact location and the icon itself.

![image](https://github.com/dotnet/docfx/assets/4528464/5f09ed41-7196-47fa-9ea0-4ec401dc4fda)

**Describe alternatives you've considered**
Another location for the opt-in icon could be on the right-hand side, above the "In this Article" section, although I am not completely certain about this placement.

**Additional context**
Adding an "Edit" button at the top of the documentation could offer increased convenience for readers who wish to contribute changes.

**Comment by yufeih** [comment]
Turn this into a discussion for more opinions from the community.

**Reply by VaclavElias** [reply]
Thanks! I hope we can get some feedback on this or more suggestions.

**Comment by VaclavElias** [comment]
Hey community, please give us some opinions 🙂
